charts/signalen-frontend/templates/configmap.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "signals-frontend.fullname" . }}
+  labels: {{- include "signals-frontend.labels" . | nindent 4 }}
+data:
+  app.json: |-
+    {{ .Values.config | toJson | nindent 4 }}
```

Comment:
Here we also need a way to override `/app/public/assets/theme.css`. We are adding this here: https://github.com/delta10/signalen-frontend-wcag/compare/main...feat/441-multiple-municipality